### DISCUSSION TITLE
Using input_select

### DISCUSSION THREAD

**Discussion by amkochaki** [ExperienceLovelace/ha-floorplan#95] [General]
I am in need of help. I need to create a button on the flooplan it would trigger an "input_select". This button would access one of inpu_select's options.
For example. In the input_select I have 3 options: Kitchen, bedroom, bathroom. I need to create a button that chooses the “fourth” option in the input_select. From that he would do an action. Would you like to make this input_select in flooplan? I already created it but I'm not getting it to work.

**Comment by OzGav** [comment]
I am not 100% sure I understand what you are trying to do but if you create a button with the input select ID and add that entity to the rules but do NOT specify a tap action then tapping on it should have the more info dialog box pop up. You should then be able to make a selection.

Make sure you have the default actions specified.

```
    defaults:
      hover_action: hover-info
      tap_action: more-info

```
